reservant-frontend/src/components/RegisterPage/Register/Register.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import RegisterForm from '../RegisterForm/RegisterForm';
+
+interface UserProps {
+    name: string,
+    surname: string,
+    password: string,
+    birthDate: string,
+    email: string,
+    phoneNumber: string
+}
+
+interface RegisterProps {
+    updateStatus: () => void
+}
+
+export default function Register ({updateStatus}: RegisterProps) {
+    
+    const onSubmit = (user: UserProps, link: string) => {
+        fetch(link, {
+            method: 'POST',
+            body: JSON.stringify({
```

Comment:
nazwy pól w obiekcie powinny pokrywać się z tymi opisanymi w końcówkach api

Reply:
tak samo w innych miejscach

Reply:
a tak dodatkowo to tutaj wystarczy podać user, nie trzeba tworzyć nowego obiektu